configure.ac

```diff
@@ -203,6 +203,12 @@ AC_ARG_ENABLE([tcti-pcap],
             [enable_tcti_pcap=yes])
 AM_CONDITIONAL([ENABLE_TCTI_PCAP], [test "x$enable_tcti_pcap" != xno])
 
+AC_ARG_ENABLE([tcti-libtpms],
+            [AS_HELP_STRING([--disable-tcti-libtpms],
+                            [don't build the tcti-libtpms module])],,
+            [enable_tcti_libtpms=yes])
```

Comment:
should we default to "check" and have the test on libtpms determine this ?

Reply:
Hm, I'm not sure. We do not check for the simulator binaries either (at least when we decide if we build the tcti modules). I though about emitting a configure warning if libtpms is not installed, but most users wouldn't and shouldn't care about that, anyway. I think I'd rather have it enabled by default... Usually it doesn't hurt and if someone wants it disabled, they can.

Reply:
My thing is: Do we want to pull in the dependency on libtpms into the build per default. Since we use the headers, we would.
For tcti-mssim/swtpm we don't do so.